cli/commander/fake-names-generator/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "fake-names-generator",
+  "version": "1.0.0",
+  "description": "CLI built with [commander](https://www.npmjs.com/package/commander) to generate fake names using [ChanceJS](https://www.npmjs.com/package/chance)",
```

```suggestion
CLI built with [commander](https://www.npmjs.com/package/commander) to generate fake names using [Chance](https://www.npmjs.com/package/chance)
```

Looking around their README, it seems the only time `ChanceJS` is in the domain. Let's change it to `Chance`, which is how they describe it in prose.